dev/src/main/resources/messages/ui/CodewindUIBundle.properties

```diff
@@ -369,3 +369,5 @@ CodewindStarted=Codewind has started. To continue, click Next.
 CodewindNotInstalled=Codewind images must be installed in order to create projects or modules.
 InstallCodewind=Install Codewind Images
 CodewindInstalledStarted=Codewind has been installed and has started. To continue, click Next.
```

Comment:
Revision:
Codewind has been installed and has started. To continue, click **Next**. 

Reason:
Make button names bold.